cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/configurations/UpdateStatusConfiguration.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.cse.import_runner.app.configurations;
+
+import com.farao_community.farao.cse.import_runner.app.util.UpdateStatusProducer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class UpdateStatusConfiguration {
+
+    @Bean("notification-producer")
```

Comment:
Update notificationProducer if you agree to change

Reply:
renamed the bean to notificationProducer